didx-xyz/yoma-mobile
# YOMA-597 - feat(CompletedChallenges): save form data

## Scope [YOMA-597](https://yomaworld.atlassian.net/browse/YOMA-597)

![](https://media.giphy.com/media/3o6MbszN7hU4LUSHyE/giphy.gif)

## Work Done
- added middleware and actions to handle sending form data to api, and handle response, saving the returned credential in state.
- renamed/refactored a few things

## Steps to Test
- run tests
- add a challenge - ensure it adds
